interface-definitions/load-balancing-inbound-ipvsadm.xml.in

```diff
@@ -0,0 +1,227 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="load-balancing">
+    <properties>
+      <help>Configure load-balancing</help>
+      <priority>900</priority>
+    </properties>
+    <children>
+      <node name="inbound">
+        <properties>
+          <help>Load balancing of inbound servers</help>
+        </properties>
+        <children>
+          <node name="ipvsadm" owner="${vyos_conf_scripts_dir}/load-balancing-inbound-ipvsadm.py">
+            <properties>
+              <help>Linux Virtual Server</help>
+            </properties>
+            <children>
+              <tagNode name="rule">
+                <properties>
+                  <help>Backend server load balancing rules</help>
+                  <valueHelp>
+                    <format>u32:</format>
```

Comment:
Invalid `format` specifier, must be `u32:0-65535`

Reply:
done